src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -199,5 +209,13 @@ private void CheckIfAnonymizedExportIsEnabled()
                 throw new RequestNotValidException(string.Format(Resources.OperationNotEnabled, OperationsConstants.AnonymizedExport));
             }
         }
+
+        private static void CheckOutputFormat(string outputFormat)
+        {
+            if (!(string.IsNullOrWhiteSpace(outputFormat) || SupportedOutputFormats.Contains(outputFormat)))
```

Comment:
Just for my understanding, are we defaulting null, empty and whitespace string to the supported output parameter?

Reply:
If a query parameter isn't given by the user it defaults to null. Whitespace would also be wrong, so I'll change it to just allow null.